pkg/apis/build/v1alpha2/image_builds.go

```diff
@@ -108,7 +108,6 @@ func lastBuild(latestBuild *Build) *LastBuild {
 
 	return &LastBuild{
 		Image:   latestBuild.BuiltImage(),
-		Cache:   BuildCache{Image: latestBuild.CacheImage()},
```

Comment:
If this is not passed through. Can we remove the whole reading of the cache image completely?

Reply:
This was an oversight, I guess there are not tests

Reply:
I added it back